pymodbus/datastore/context.py

```diff
@@ -27,6 +28,45 @@ def decode(self, fx):
         """
         return self._fx_mapper[fx]
 
+    async def async_getValues(self, fc_as_hex, address, count=1):
+        """Get `count` values from datastore.
+
+        :param fc_as_hex: The function we are working with
+        :param address: The starting address
+        :param count: The number of values to retrieve
+        :returns: The requested values from a:a+c
+        """
+        return self.getValues(fc_as_hex, address, count)
+
+    async def async_setValues(self, fc_as_hex, address, values):
+        """Set the datastore with the supplied values.
+
+        :param fc_as_hex: The function we are working with
+        :param address: The starting address
+        :param values: The new values to be set
+        """
+        return self.setValues(fc_as_hex, address, values)
+
+    def getValues(self, fc_as_hex, address, count=1):
+        """Get `count` values from datastore.
+
+        :param fc_as_hex: The function we are working with
+        :param address: The starting address
+        :param count: The number of values to retrieve
+        :returns: The requested values from a:a+c
+        """
+        del fc_as_hex, address, count
```

Comment:
Why delete the variables, that does not make a lot of sense.

Reply:
they are not used, so makes it more clear in that point. Mypy and ruff etc easily complain on those. But I agree it is more of taste and not really critical to be in my point of view.

Reply:
use "_" that is the standard way to tell a variable is unused.

It might not be critical from your POW, but from me as maintainer it is negative.

Reply:
sure, I can change it to be with "_".